boardgame-ui/src/utils/api/Model.js

```diff
@@ -118,10 +118,15 @@ export default class Model {
    *
    * @returns {this} The saved object, as returned by backend
    */
-  async save() {
+  async save(params={}) {
```

Comment:
didn't know you could do that in javascript !

Reply:
credits goes to @elodieburtin :p